marketplace/commercetools-products/src/dialog.js

```diff
@@ -0,0 +1,91 @@
+/* global CTPicker */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button } from '@contentful/forma-36-react-components';
+
+export class CommerceToolsDialog extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.pickerContainerRef = React.createRef();
+    this.picker = null;
+
+    this.onCancel = this.onCancel.bind(this);
+    this.onInsert = this.onInsert.bind(this);
+  }
+
+  render() {
```

Comment:
Style suggestion: I would reorder these methods to match all the other React components. For example: putting the render method last. It's not really a blocker, just a consistency thing.

Reply:
This is one of those things I prefer not to do without a linter enforcing it. Keeping these rules in my head (when I only rarely work in repos with this code style) is a lot. If it's a rule in `@contentful/eslint-config-extension`, that'd be cool.